requirements/prod.txt

```diff
@@ -0,0 +1,34 @@
+# TODO: keep aligned with pyproject.toml
```

Comment:
agreed! the only alternative i know is just `pip install .` the whole package, but since the package's code is going to change much more often than the dependencies, it seemed like a bad way to leverage docker caching.